Ascend-Server.api/Services/WellnessRatingService.cs

```diff
@@ -0,0 +1,88 @@
+using Models; 
+using Exceptions; 
+using System; 
+namespace Services; 
+
+public static class WellnessRatingService
```

Comment:
Don't use static for services. It makes testing the interactions with this service impossible. You want to manage the instantiation of this object and use dependency injection when you have more dependencies.